packages/babel-helper-replace-supers/src/index.ts

```diff
@@ -64,6 +65,26 @@ const visitor = traverse.visitors.merge<
     Super(path, state) {
       const { node, parentPath } = path;
       if (!parentPath.isMemberExpression({ object: node })) return;
+      if (parentPath.parentPath.isUnaryExpression({ operator: "delete" })) {
```

Comment:
This logic should be moved in a different place: instead of special-casing `delete` here, we should add an optional `delete` handler to https://github.com/babel/babel/blob/65c8496b8403aecb5e8bfc580c38321021d1ca16/packages/babel-helper-member-expression-to-functions/src/index.ts#L537, and implement it in https://github.com/babel/babel/blob/65c8496b8403aecb5e8bfc580c38321021d1ca16/packages/babel-helper-replace-supers/src/index.ts#L121

Reply:
Done in 38dcf9d2b364c1d181a6b34553ef6b1d6c78c683 :)